boards/arm/frdm_k22f/dts_fixup.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Prevas A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_FXOS8700_NAME				\
+	DT_NXP_KINETIS_I2C_40066000_NXP_FXOS8700_1D_LABEL
```

Comment:
Don't worry about the 80-char limit in the fixup file. Please make these defines one line each instead of multiline